From cccc3bd297e77ac13a5da75b2a1bdeb952e84c1e Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Tue, 10 Jul 2018 14:20:57 +0200
Subject: [PATCH 18/19] Increase max aspect ratio.

Make sure the app is displayed in full-screen on devices with
18:9 and 19:9 screen.

Change-Id: I715d254e98af261ce6b881bb0c2b79eb32b4bd9c
---
 AndroidManifest.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index deb19a09a..4a592f821 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -87,6 +87,8 @@
         android:supportsRtl="true"
         android:usesCleartextTraffic="true" >
 
+        <meta-data android:name="android.max_aspect" android:value="2.5" />
+
         <uses-library android:name="org.apache.http.legacy" android:required="false" />
         <activity
             android:name="com.android.email.activity.ComposeActivityEmail"
-- 
2.17.1

